magento/magento2
# comments & validation added in cookie configuration

### Description (*)

Go to Stores -> Settings -> Configuration -> General -> Web and expand Default Cookie Settings section.
1. Added comments to each field for a better understanding of the admin user.
2. Added number validation for cookie lifetime field to make sure it only accepts numbers.

### Manual testing scenarios (*)

1. Given string characters in the cookie lifetime field and verify number validation is working
2. Verify the comments are displayed under each field of the section.

### Contribution checklist (*)
